audiomentations/augmentations/wind_absorption.py

```diff
@@ -0,0 +1,167 @@
+from audiomentations.core.transforms_interface import BaseWaveformTransform
+import numpy as np
+import librosa
+
+
+class WindAbsorption(BaseWaveformTransform):
+    """
+    Applies a wind absorption transform parametrized by temperature, humidity, and distance
+    of source.
+
+    This is not a scientifically accurate transform but basically applies a uniform
+    filterbank with attenuations given by:
+
+    att = exp(- distance * absorption_coefficient)
+
+    where distance is the microphone-source assumed distance in meters and `absorption_coefficient`
+    is adapted from a look up table by pyroomacoustics [1].
+
+    Note: This only "simulates" room absorption, and does not apply the distance law
+    (use a GainTransform for that)
```

Comment:
Note: This only "simulates" the dampening of high frequencies, and does not gain down the audio according to the distance law.
Gain augmentation needs to be done separately